.gitignore

```diff
@@ -5,4 +5,6 @@
 *.*~
 application.properties
 bin/
-.metadata/
+.metadata
+src/main/resources/conf/application.properties
```

Comment:
This doesn't help, read more here if you're interested http://source.kohlerville.com/2009/02/untrack-files-in-git/